cmd/solana_exporter/exporter.go

```diff
@@ -72,27 +72,31 @@ func NewSolanaCollector(rpcAddr string) *solanaCollector {
 func (c *solanaCollector) Describe(ch chan<- *prometheus.Desc) {
 	ch <- c.totalValidatorsDesc
 	ch <- c.solanaVersion
+	ch <- c.validatorActivatedStake
```

Comment:
Not sure why these weren't here originally, but the tests were failing, claiming that the descriptors weren't registered, only fix I found was adding these lines. 

There is also the `prometheus.DescribeByCollect` function (see docs [here](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#DescribeByCollect) which also seems to work in this case, however I don't understand the system well enough to know which is better for now.